my-app/src/App.js

```diff
@@ -1,10 +1,25 @@
-import './App.css';
-import Header from './components/Header';
-
+import "./App.css";
+import { useEffect, useState } from "react";
+import Header from "./components/Header";
+import ContentList from "./components/ContentList";
+require("dotenv").config();
 function App() {
+  const [content, setContent] = useState();
+  useEffect(() => {
+    const apikey = process.env.REACT_APP_UNSPLASH_KEY;
```

Comment:
Imagine this scenario; that your dependency array of your `useEffect` is not empty/ has a state variable, which means this function will be invoked whenever this state changes, right? ya3ni, it will read this `const` value on each render. However,  `REACT_APP_UNSPLASH_KEY` value will be static here, and reading it for once would be enough for you, you don't need to keep reading it on each render. So it will be better if you moved this line out of your hook here.

Reply:
@Jood80 But that's the point of having an empty dependency array, it would've been wrong if it wasn't empty, but it's alright the way it is.